### PR TITLE
doc(profiles): Add deploy profiles app doc

### PR DESCRIPTION
**Pull request by lutangar**

### CODE REVIEW

File: docs/CONTRIBUTING.md [hunk 1]
```diff
@@ -186,6 +186,36 @@ Once the PR is merged into `master` a build is triggered on [SemaphoreCI](https:
...
+> ```
+
+1. Deployment on __Clever Cloud__ is done via git :
+```
+git remote add production-profiles git+ssh://git@push-n2-par-clevercloud-customers.services.clever-cloud.com/app_b47e0ed5-d153-4cc4-a306-28fe156c5cee.git
+git checkout master
+git push production-profiles HEAD
+```
+
+2. You may then update the Js bundle version in the Divi theme options, to force cache refresh:
```
Comment (gregoirelacoste) [inline docs/CONTRIBUTING.md:215]: It seems working without doing that

File: docs/CONTRIBUTING.md [hunk 2]
```diff
@@ -186,6 +186,36 @@ Once the PR is merged into `master` a build is triggered on [SemaphoreCI](https:
...
+#### Staging & proding
+
+`staging`and `proding` environment are automatically deployed on __Clever Cloud__ on a new push on the `master` branch:
+
+- https://staging-profiles.dismoi.io/
+- https://proding-profiles.dismoi.io/
+
+#### Production
+
+DisMoi `production` profiles app is available at https://www.dismoi.io/eclaireurs/.
```
Comment (gregoirelacoste) [inline docs/CONTRIBUTING.md:200]: Prod version is first in profiles.dismoi.io (corvée cloud) and then Linked in dismoi.io

File: docs/CONTRIBUTING.md (hunk 1 above)
Comment (lutangar) [reply docs/CONTRIBUTING.md:215]: Yep it works without doing, that's why I'm saying *you may*. But it forces the cache refresh so the rest of the world don't have to clear cache.

File: docs/CONTRIBUTING.md (hunk 2 above)
Comment (lutangar) [reply docs/CONTRIBUTING.md:200]: > corvée cloud

:rofl: 

That's what I tried to express in the second paragraph, but apparently not clear enough. What's your suggestion?

Comment (JalilArfaoui) [reply docs/CONTRIBUTING.md:200]: Clear enough for me (about Corvée Cloud :-) )

**Review by JalilArfaoui — APPROVED** [review]

File: docs/CONTRIBUTING.md (hunk 2 above)
Comment (gregoirelacoste) [reply docs/CONTRIBUTING.md:200]: corvée cloud :-) thx to smartphone

**Review by gregoirelacoste — APPROVED** [review]

**Pull request merged**